tests/mappers/test_axisymmetric_3d_to_2d.py

```diff
@@ -15,12 +15,20 @@ def setUp(self):
                            'settings':
                                {'direction_axial': 'x',
                                 'direction_radial': 'y',
-                                'n_tangential': 13}
+                                'angle': 180,
+                                'n_tangential': 9}
                            }
 
     def test_initialize(self):
         mp_name_in = 'wall_in'
         mp_name_out = 'wall_out'
+        key = 'angle'
+        if key in self.parameters['settings']:
+            self.angle = self.parameters['settings']['angle']
+            print(key + ' is set on ' + str(self.angle) + ' degrees')
+        else:
+            self.angle = 360
+            print(key + ' is set on ' + str(self.angle) + ' degrees')
```

Comment:
Modify this to angle = self.parameters['settings'].get('angle', 360) (I would not make angle a class variable, as you only use it locally)